openapi/components/schemas/PayoutRequest.yaml

```
@@ -93,6 +93,13 @@ properties:
             - $ref: ./ServerTimestamp.yaml
         updatedTime:
           $ref: ./UpdatedTime.yaml
+  selectedPaymentInstrumentRedirectUrl:
+    description: |-
+      URL to redirect the customer to when payment instrument is selected. The default value is the website URL.
+      `{{id}}` can be used as a placeholder for the payout request ID.
```

suggestion: suggestion to avoid 'can be' (may sound passive)
```suggestion
      Use `{{id}}` as a placeholder for the payout request ID.
```